docs/software/python-cli/installation.mdx

```diff
@@ -163,7 +163,7 @@ values={[
 - Check that your computer has Python 3 installed.
   - Use the command
     ```powershell
-    py -V
+    python -V
```

Comment:
My understanding is that `py` is available by default when installing on Windows. If it is not working there is either a problem with the installation/path or the install was customized in some way.

Reply:
yes, I use virtual env for python.